pages/tabs/integration-test.page.tsx

```diff
@@ -58,7 +58,11 @@ export default function TabsScenario() {
       <h1>Tabs integration test page</h1>
       <ScreenshotArea disableAnimations={true}>
         <div id="click-this" tabIndex={0} style={{ height: 10 }} />
-        <Tabs id="first-tabs" tabs={tabs} />
+        <Tabs
+          id="first-tabs"
+          tabs={tabs}
+          i18nStrings={{ scrollLeftAriaLabel: 'Scroll left', scrollRightAriaLabel: 'Scroll right' }}
+        />
         <div id="click-this-2" tabIndex={0} style={{ height: 10 }} />
 
         <Tabs tabs={tabs} variant="container" />
```

Comment:
missing i18nStrings here

Reply:
Right, will add them. The prop is not mandatory because of backwards compatibility, but I think it's good to add them to all test pages to make them future-proof.